pennylane/_device.py

```diff
@@ -206,9 +206,7 @@ def execute(self, queue, observables, parameters={}, **kwargs):
             self.pre_apply()
 
             for operation in queue:
-                # indices of the wires in the device's register
```

Comment:
So now up to devices to implement the translation of wires to registers/subsystems?

Reply:
Yep!